doc/source/user_guide/fields/field_data.rst

```diff
@@ -100,7 +100,7 @@ To retrieve scalar field data, such as absolute pressure, use ``ScalarFieldDataR
 
   >>> from ansys.fluent.core.services.field_data import ScalarFieldDataRequest
   >>> scalar_request_1 = ScalarFieldDataRequest(field_name="absolute-pressure", surfaces=["inlet"])
-  >>> abs_press_data = field_data.get_field_data(scalar_request_1)
+  >>> abs_press_data = field_data.get_field(scalar_request_1)
```

Comment:
@prmukherj I would prefer to use more indicative variable names such as `pressure_request` and `pressure_data`. It makes it easier to follow the workflow. `scalar_request_1` does not carry much explicit meaning while `abs_press_data` contains two abbreviations which directly impedes clarity.

Reply:
Done, Thank you.